src/components/MainContent/index.jsx

```diff
@@ -24,7 +24,7 @@ const MainContent = (props) => {
             <Switch>
               <Route exact path="/" component={IndexPageContainer} />
               <PrivateRouteContainer path="/search" component={SearchPageContainer} />
-              <PrivateRouteContainer path="/preview/:previewId" component={JournalPreviewContainer} />
+              <PrivateRouteContainer path="/:journalAboutId/preview/:previewId" component={JournalPreviewContainer} />
```

Comment:
Since we're using journalAboutId, would it make sense to add it to the JournalRouterContainer?

Reply:
that makes sense. I moved it to JournalRouterContainer